kafka-ui-api/src/main/java/com/provectus/kafka/ui/strategy/ksql/statement/BaseStrategy.java

```diff
@@ -16,15 +16,15 @@
 import java.util.stream.StreamSupport;
 
 public abstract class BaseStrategy {
-  protected static final String KSQL_REQUEST_PATH = "/ksql";
-  protected static final String QUERY_REQUEST_PATH = "/query";
+  protected static final String KSQL_REQUEST_PATH = "ksql";
+  protected static final String QUERY_REQUEST_PATH = "query";
   private static final String MAPPING_EXCEPTION_ERROR = "KSQL DB response mapping error";
   protected String host = null;
   protected KsqlCommandDTO ksqlCommand = null;
 
   public String getUri() {
     if (this.host != null) {
-      return this.host + this.getRequestPath();
+      return this.host + "/" + this.getRequestPath();
```

Comment:
Is that a thing sonar is whining about? Such a weird case tbh

Reply:
Yes, it,s java:S1075 (URIs should not be hardcoded). The problem was in the `KSQL_REQUEST_PATH = "/ksql"; QUERY_REQUEST_PATH = "/query"` constants that stored the "/". Sonar does not recommend doing this